src/extension/rn-extension.ts

```diff
@@ -1,6 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
+import * as fs from "fs";
+
+try {
+    fs.statSync(`${__filename}.map`); // We check if source maps are available
```

Comment:
It won't. Line 7 is not for Production, but for us, in case we are ever debugging the extension without source maps for some reason.